vars/cloudFoundryDeploy.groovy

```diff
@@ -164,7 +165,7 @@ void call(Map parameters = [:]) {
                 deploy = true
                 // set default mtar path
                 config = ConfigurationHelper.newInstance(this, config)
-                    .addIfEmpty('mtaPath', config.mtaPath?:findMtar())
+                    .addIfEmpty('mtaPath', config.mtaPath?:findMtar(config))
```

Comment:
Spreading the config around into auxiliar functions just for the sake of logging is maybe not that cool. I would think twice before doing this. It is not clear if the config is really only used for logging purposes or if the configuration is evaluated downwards the road. And if the config is present in the auxiliary functions it might get used there lateron instead of retrieving config values at a higher level and just pass it as parameters into the auxiliary functions. Basically we will end up in code harder to read and to understand when having the config everywhere (and config evaluated at several levels (inside auxiliary methods and inside higher level methods, e.g. `call`).

This applies also downwards in this PR.